lib/packager/Packager.cpp

```diff
@@ -48,6 +48,7 @@ namespace MAT_NS_BEGIN {
             LOG_TRACE("Adding event %s:%s, size %u bytes",
                 tenantTokenToId(record.tenantToken).c_str(), record.id.c_str(), static_cast<unsigned>(record.blob.size()));
 
+            ctx->cvStr = record.cvStr;
```

Comment:
Why is it `cvStr` ? Correlation Vector technology is no longer recommended for use. This is supposed to be W3C TraceContext going forward. I think even naming this thing using a "legacy tech" name isn't great..

Reply:
https://www.w3.org/TR/trace-context/

Reply:
+1. If this code ever does get integrated back into the main branch, it would be good to use agreed upon standards. Since this is a private branch, I'm fine with this for now, but it would be best to update it to conform to the standard.

Reply:
Thanks @maxgolov let me have a look. Sure Ryan we will update it as required.

Updated description with design doc, @maxgolov you can refer.  Basically, this required for MIP as per customer issue diagnostics.

Just mentioning the option we already tried and stuck  later:

- we tried integrating the client telemetry(1DS) Static library into the MIP code for integrating debug event listener from the MIP code

but it's generating a Runtime library issue for MIP build since MIP code uses CXXFlags MDd for using dll versions. 
Changing these flags in the MIP code may require our partners to also update these CXXFlags as well, hence this change will be very sensitive for us and risky. We are looking into it parallelly.

- 1DS Static library size is also concerning ~150 MB whereas dynamic library size if ~4 MB.

- On the MIP side DoSendHttp method, we tried to decode the request body sent by one DS in the attached callback. But that also showed a linker issue since the symbol was not exposed.


Please let us know if there is some easy way to achieve like mentioned below or some other:

- Exposing required function from client telemetry dll version.
- Passing custom logger delegate and unique id to 1DS.